documentation/site/pages/zkc/mining/walkthrough.mdx

```diff
@@ -4,53 +4,65 @@
+The resultant state file keeps track of all work done so far for one reward address. Each reward address should have *exactly* one state file associated with it. Submitting work for a rewards address requires all previously submitted receipts to be included in the state file. *The same state file must be used for submitting any additional work done by the associated rewards address.* If two state files have the same rewards address, they will conflict during `submit-mining` because they will have different Merkle tree states.
 
 :::warning
-Please note that this state file is *very important* and it should be kept in a durable location. 
+Please note that this state file is *very important* and it should be kept in a durable location.
 
-Loss of this state file *will result* in the loss of all work that is not submitted for the respective rewards address, meaning rewards for any work done since the last `submit` **will be lost**. 
+Loss of this state file *will result* in the loss of all work that is not submitted for the respective rewards address, meaning rewards for any work done since the last `submit-mining` **will be lost**.
 
 In this scenario, a new rewards address must be specified for Bento, and Bento should be restarted to allow further work to be recorded properly against the new rewards address. *To avoid this scenario, it is recommended to store the state file in a durable location.*
 :::
 
-Provers only specify the `--new ${REWARD_ADDRESS}` flag once, this is necessary to create the state file for that specific rewards address. Once the state file is created, provers can run `prepare` and use the state file like so:
+Provers only specify the `--new ${REWARD_ADDRESS}` flag once, this is necessary to create the state file for that specific rewards address. Once the state file is created, provers can run `prepare-mining` and specify the state file location directly:
 
 ```bash
-boundless povw prepare --state ${STATE_FILE_LOCATION} --from-bento
+boundless rewards prepare-mining --state-file ${STATE_FILE_LOCATION} --from-bento
 ```
 
-## Submit the aggregated work proof onchain
+or add the mining state file location to the rewards module config by running:
 
-After running `prepare`, provers are ready to submit onchain with:
+```bash
+boundless rewards setup
+```
+
+which would store the state file location, such that provers can simply run:
 
 ```bash
-boundless povw submit --state ${STATE_FILE_LOCATION} --rpc-url ${RPC_URL} --private-key ${PRIVATE_KEY} --povw-private-key ${POVW_PRIVATE_KEY}
+boundless rewards prepare-mining --from-bento
```

Comment:
i removed --from-bento as it made some assumptions on where you are running the command,

`--bento-api-url <URL>`